k/kong/2.3.4/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM registry.access.redhat.com/ubi7/ubi AS build
```

Comment:
I know we started here with a existing Dockerfile but just wanted to confirm if there is a specific requirement for ubi 7 here. if not might be better to go for ubi 8.x (preferably) 8.7?